misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java

```diff
@@ -46,6 +46,13 @@ public class Messages {
 
     private static final String NO_BALLERINA_FILE_OR_MODULE = "couldn't find an existing ballerina file or " +
             "module by the name: ";
+
+    private static final String CANT_ALLOW_BOTH_MODULE_AND_FILE_OPTIONS =
+            "too many options: either use module option or file option";
+
+    private static final String CANT_ALLOW_MODULE_OR_FILE_OPTIONS =
+            "too many options: single file projects cannot accept module or file option";
```

Comment:
```suggestion
    private static final String CANT_ALLOW_MODULE_OR_FILE_OPTIONS =
            "'--module' and '--file' options are not allowed on single file projects";
```

Reply:
@praneesha WDYT?

Reply:
How about `Do not use '--module' and '--file' options in single file projects.`?

Reply:
@praneesha these are error messages and, IINM we haven't used this format for the CLI error messages in other context? ( Correct me if I'm wrong :) )

Reply:
yes. +1 for `"The '--module' and '--file' options are not allowed in single file projects";`